doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -168,6 +168,20 @@ Additional scenario XML elements:
   Specify whether force to disable software workaround for Machine Check
   Error on Page Size Change is enabled.
 
+``IVSHMEM`` (a child node of ``FEATURE``):
+  Specify the IVSHMEM configuration
+
+``IVSHMEM_ENABLED`` (a child node of ``FEATURE/IVSHMEM``):
+  Specify whether Enable Share Memory between VMs by IVSHMEM.
+
+``IVSHMEM_REGION`` (a child node of ``FEATURE/IVSHMEM``):
+  Specify the name, size, IDs of communication VMs of share memory region,
```

Comment:
Let's change the description to:
```
  Specify a comma-separated list of the inter-VM shared memory region name,
  size, and VM IDs that may communicate using this shared region.

  * Prefix the region ``name`` with either ``hv:/`` (for an hv-land solution) or
    ``dm:/`` for a dm-land solution; for example, ``hv:/shared01``.
    (See :ref:`ivshmem-hld` for details.)
  * Specify the region ``size`` in MB, and a power of 2 (e.g., 2, 4, 8, 16)
    up to 512.
  * Specify the two VM IDs that may use this shared memory area,
    separated by a ``:``, for example, ``0:2`` 
```
@xiaofst Your syntax description seems to allow you to share the memory area between more than two regions by writing this as, for example, ``hv:/myname,16,0:1,0:2:1:2``, but I don't think that's allowed.  Can you confirm?

Reply:
@dbkinder , synced with IVSHMEM feature owner @liuyuan-GitHub ,can we use below description:
  * Specify the VM IDs that may use this shared memory region,
     separated by ``:``, for example, ``0:2``.

The format should be hv:/<shm name>,<shm size in MB>,<vm id:vm id[:vm id:...]> . For example, hv:/myname, 16, 0:1  or hv:/myname, 16, 0:1:2